packages/web/docs/src/pages/docs/other-integrations/apollo-router.mdx

```diff
@@ -249,6 +251,71 @@ plugins:
     # accept_invalid_certs: true
 ```
 
+## App Deployments (Persisted Documents)
+
+To activate [App Deployments]() in your Apollo Router, you need to enable the
+`hive.persisted_documents` plugin. The plugin uses the `HIVE_CDN_ENDPOINT` and `HIVE_CDN_KEY`
+environment variables to resolve the persisted documents IDs.
+
+```yaml filename="router.yaml"
+# ... the rest of your configuration
+plugins:
+  hive.usage: {}
+  hive.persisted_documents:
+    enabled: true
+```
+
+Once enabled, follow
+[App Deployments guide](/docs/schema-registry/app-deployments#sending-persisted-document-requests-from-your-app)
+guide to send persisted document requests from your app, using the Hive CDN for resolving the
+document IDs.
+
+### Configuration
+
+The following environment variables are required:
+
+- `HIVE_CDN_ENDPOINT` - the endpoint Hive generated for you in the previous step (for example:
+  `https://cdn.graphql-hive.com/artifacts/v1/TARGET_ID/supergraph`)
+- `HIVE_CDN_KEY` - the access key to Hive CDN
+
+You may configure the plugin with the following options:
+
+```yaml filename="router.yaml"
+# ... the rest of your configuration
+plugins:
+  hive.usage: {}
+  hive.persisted_documents:
+    # Enables/disables the plugin
+    # Default: true
+    enabled: true
+    # The endpoint of the Hive CDN. You can either specify it here, or use the existing HIVE_CDN_ENDPOINT environment variable.
+    # Required
+    endpoint: 'https://cdn.graphql-hive.com/artifacts/v1/TARGET_ID/supergraph'
+    # The access key to the Hive CDN. You can either specify it here, or use the existing HIVE_CDN_KEY environment variable.
+    # Required
+    key: '...'
+    # Enables/disables the option to allow arbitrary documents to be executed.
+    # Optional
+    # Default: false
+    allow_arbitrary_documents: false
+    # HTTP connect timeout in seconds
+    # Optional
+    # Default: 5
+    connect_timeout: 5
+    # HTTP request timeout in seconds
+    # Optional
+    # Default: 15
+    request_timeout: 15
+    # Accepts invalid SSL certificates
+    # Optional
+    # Default: false
+    accept_invalid_certs: false
```

Comment:
I assume these are related to resolving the documents from the CDN? If so it should probably be noted there.